assembl/static2/package.json

```diff
@@ -96,27 +97,27 @@
     "piwik-react-router": "assembl/piwik-react-router#skip_piwik_initialization",
     "prop-types": "^15.5.8",
     "rc-slider": "^8.6.0",
-    "react": "^15.4.1",
+    "react": "^16.3.1",
```

Comment:
What kind of impact will this have on `react-virtualized` ? I can imagine that their package will either benefit greatly from this, which in turn, might mean breaking change upgrade for use (hopefully not).

Reply:
I don't see the relation with react-virtualized. Are you talking about async rendering? This is not currently enabled in react 16.3, it will be in react 17. There is lots of fixes to do on deprecated api before we can upgrade to a react 17.
https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html